confluent_platform/datadog_checks/confluent_platform/data/metrics.yaml

```diff
@@ -0,0 +1,1276 @@
+          metric_type: gauge
+
+  # Kafka Broker Server Metrics Regex (Yammer Metric Gauge)
+  - include:
+      domain: kafka.server
+      bean_regex:
+        # Lag in number of messages per follower replica. This is useful to know if the replica is slow or has stopped replicating from the leader.
+        - kafka\.server:type=FetcherLagMetrics,name=ConsumerLag,.*
+      attribute:
+        Value:
+          alias: $domain.fetcher_lag.$name
+          metric_type: gauge
+
+  # Kafka Broker Cluster Metrics Regex (Yammer Metric Gauge)
+  - include:
+      domain: kafka.cluster
+      bean_regex:
+        # Number of partitions whose in-sync replicas count is less than minIsr. These partitions will be unavailable to producers who use acks=all.
+        - kafka\.cluster:type=Partition,name=UnderMinIsr,.*
+      attribute:
+        Value:
+          alias: $domain.$type.$name
+          metric_type: gauge
+
+  # Kafka Broker Topic Metrics (Yammer Metric Meter)
+  - include:
+      domain: kafka.server
+      bean:
+        # Aggregate incoming byte rate.
+        - kafka.server:type=BrokerTopicMetrics,name=BytesInPerSec
+        # Aggregate outgoing byte rate.
+        - kafka.server:type=BrokerTopicMetrics,name=BytesOutPerSec
+        # Aggregate rejected byte rate.
+        - kafka.server:type=BrokerTopicMetrics,name=BytesRejectedPerSec
+        # Fetch request rate for requests that failed.
+        - kafka.server:type=BrokerTopicMetrics,name=FailedFetchRequestsPerSec
+        # Produce request rate for requests that failed.
+        - kafka.server:type=BrokerTopicMetrics,name=FailedProduceRequestsPerSec
+        # Aggregate incoming message rate.
+        - kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec
+        # Fetch request rate.
+        - kafka.server:type=BrokerTopicMetrics,name=TotalFetchRequestsPerSec
+        # Produce request rate.
+        - kafka.server:type=BrokerTopicMetrics,name=TotalProduceRequestsPerSec
+      attribute:
+        Count:
+          alias: $domain.topic.$name.rate
+          metric_type: rate
+
+  # Kafka Broker Topic Metrics Regex (Yammer Metric Meter)
+  - include:
+      domain: kafka.server
+      bean:
+        # Rate at which the pool of in-sync replicas (ISRs) expands.
+        - kafka.server:type=ReplicaManager,name=IsrExpandsPerSec
+        # Rate at which the pool of in-sync replicas (ISRs) shrinks.
+        - kafka.server:type=ReplicaManager,name=IsrShrinksPerSec
+      attribute:
+        Count:
+          alias: $domain.$type.$name.rate
+          metric_type: rate
+  - include:
+      domain: kafka.server
+      bean:
+        # Average fraction of time the request handler threads are idle. Values are between 0 (all resources are used) and 1 (all resources are available)
+        - kafka.server:type=KafkaRequestHandlerPool,name=RequestHandlerAvgIdlePercent
+      attribute:
+        Count:
+          alias: $domain.request_handler_pool.avg_idle_percent.rate
+          metric_type: rate
```

Comment:
Not sure why a percentage is submitted as a rate, so ditto

Reply:
Answered here https://github.com/DataDog/integrations-core/pull/5733#discussion_r392942637